### PR TITLE
Auth context

### PR DESCRIPTION
**Pull request by azaroth42**
* Add @context to auth and all examples
* Update image 2.1 to use auth
* Update description of status codes for auth errors
* Add window closing to 2.1.2

### CODE REVIEW

File: source/api/auth/0.9/index.md [hunk 1]
```diff
@@ -93,17 +94,17 @@ The Description Resource _MUST_ include a service using the following template:
 }
 {% endhighlight %}
 
-Where the `@id` field _MUST_ be present and contains the URI that the client should load to allow the user to authenticate. The value of `profile` _MUST_ be `http://iiif.io/api/auth/{{ page.major }}/login`, allowing clients to understand the use of the service. The `label` property _SHOULD_ be present and its value is to be shown to the user to initiate the loading of the authentication service.  The `service` field _MUST_ be present and will contain related services described below.
+Where the `@id` field _MUST_ be present and contains the URI that the client should load to allow the user to authenticate. The `@context` field _MUST_ be present with the value `http://iiif.io/api/auth/{{ page.major }}/context.json`, specifying the context to resolve the keys into RDF if necessary. The value of `profile` _MUST_ be `http://iiif.io/api/auth/{{ page.major }}/login`, allowing clients to understand the use of the service. The `label` property _SHOULD_ be present and its value is to be shown to the user to initiate the loading of the authentication service.  The `service` field _MUST_ be present and will contain related services described below.
```
Comment (jpstroop) [inline source/api/auth/0.9/index.md]: "should load to allow" -> "should load in order to allow"??

File: source/api/auth/0.9/index.md [hunk 2]
```diff
@@ -170,7 +172,7 @@ Authorization: Bearer TOKEN_HERE
 
 #### 2.2.3. JSONP Interaction
 
-Browser based clients _MUST_ use [JSONP][jsonp] callbacks to retrieve the access token.  The request _MUST_ have a `callback` parameter added to the URL from the `@id` field, and if an authorization code is required, that _MUST_ be present in a `code` parameter.  An example URL:
+Browser based clients _MUST_ use [JSONP][jsonp] callbacks to retrieve the access token; see below for the rationale for this.  The request _MUST_ have a `callback` parameter added to the URL from the `@id` field, and if an authorization code is required, that _MUST_ be present in a `code` parameter.  An example URL:
```
Comment (jpstroop) [inline source/api/auth/0.9/index.md]: "see below" where? Can this be a link or a more specific forward reference?

File: source/api/image/2.1/index.md [hunk 3]
```diff
@@ -637,7 +637,7 @@ More complex examples are given in the [Complete Response Example](#complete-res
 
 There _MAY_ be one or more services associated with an image. See the [Service Profiles][service-profiles] annex for more information.
 
-The following shows a simple use of `service` to associate geographic metadata with the image.
+The following shows a use of `service` to associate the login page of an authentication system needed to access the image.  For further information, please see [Authentication](#authentication).
```
Comment (jpstroop) [inline source/api/image/2.1/index.md]: "the image" -> "an image"

File: source/api/auth/0.9/index.md (hunk 1 above)
Comment (azaroth42) [inline source/api/auth/0.9/index.md]: Sure. Some people have said I use too many small but verbose constructions like this, but if it makes it clearer...

File: source/api/image/2.1/index.md (hunk 3 above)
Comment (azaroth42) [inline source/api/image/2.1/index.md]: It's the image as described in the current info.json response. All of the other 5.x sections use "the image" too. Will try to make it clearer though.

**Pull request merged**